eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala

```diff
@@ -48,12 +49,14 @@ object ExplorerApi {
   implicit val formats: DefaultFormats = DefaultFormats
   implicit val serialization: Serialization = Serialization
 
+  implicit def scala2kmp(input: ByteVector32): fr.acinq.bitcoin.ByteVector32 = new fr.acinq.bitcoin.ByteVector32(input.toArray)
+
   sealed trait Explorer {
     // @formatter:off
     /** Explorer friendly-name. */
     def name: String
     /** Map from chainHash to explorer API URI. */
-    def baseUris: Map[ByteVector32, Uri]
+    def baseUris: Map[fr.acinq.bitcoin.ByteVector32, Uri]
```

Comment:
Can we use a `ByteVector32` here instead? It would be nice to keep the `fr.acinq.bitcoin.ByteVector32` hidden away as much as possible, wouldn't it?